### DISCUSSION TITLE
How to provide private access to embargoed dataset

### DISCUSSION THREAD

**Discussion by Thart92** [dandi/dandi-archive#1817] [Q&A]
Hi, 

I have uploaded my unpublished data, which is currently set to embargoed status. Before the associated paper is published, I would like to provide access to collaborators and reviewers via a link. However, the link generated by the "share" button seems not to work; ie when others click it they cannot access the dataset. How can I provide access to necessary parties while maintaining embargoed status on the repository?

**Comment by CodyCBakerPhD** [comment]
Hi there @Thart92!

Access to embargoed assets is achieved by adding the individuals you want to share the dataset with as additional 'owners'; this technically gives those other individuals write permissions as well

Adding alternative levels of permissions is a feature the DANDI team has received in the past (such as 'view only'); assuming you'd prefer to add certain individuals as 'read-only' permissiveness, I'd be happy to bring it up to the team again to see what the status is on adding this feature

**Reply by Thart92** [reply]
Thank you for the answer! I agree that it would be great to have the option to add 'read-only' permissions. It would also be very useful to be able to designate a dataset as hidden (not findable by search, not listed publicly) but viewable to someone with the link. Then it could be easily made available to anonymous reviewers prior to making it available to the general public.

**Reply by satra** [reply]
we are in active discussions for future of access control in dandi. would you want finer granularity? for example the reviewer can see but not download, or is a simple level of read-only vs read-write sufficient for your needs?

**Reply by Thart92** [reply]
For my purposes, I would like to grant download access, so the raw data can be thoroughly examined if the reviewer desires.
